libs/ui-components/src/components/Device/EditDeviceWizard/steps/DeviceTemplateStep.tsx

```diff
@@ -21,7 +25,62 @@ const templateOption1 = '{{ device.metadata.labels[key] }}';
 const templateOption2 = '{{ device.metadata.name }}';
 const exampleCode = `/device-configs/factory-floors/floor-{{ device.metadata.labels[factory-floor] }}`;
 
+const MicroShiftCheckbox = ({ isFleet }: { isFleet: boolean }) => {
+  const { t } = useTranslation();
+
+  const [repo, loading, error] = useFetchPeriodically<Required<Repository>>({
+    endpoint: `repositories/acm-registration`,
```

Comment:
Would be better to define the repository name in a constant.

Reply:
constant is created, but not used here :) fixed